### PR TITLE
Fix Home Page Fabs

### PR DESCRIPTION
**Pull request by leeminkyu16**
Shows fab on map view and listing view while loading

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt [hunk 1]
```diff
@@ -73,159 +75,159 @@ fun HomeListChildView(
...
+				mutableStateOf(false)
+			}
+			val listState: LazyListState = rememberLazyListState()
+			val lastItemIsShowing by remember {
+				derivedStateOf {
+					if (listState.layoutInfo.totalItemsCount == 0) {
+						false
+					} else {
+						listState.layoutInfo.visibleItemsInfo.lastOrNull()?.index == listState.layoutInfo.totalItemsCount - 1
+					}
```
Comment (andr-dev) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt:135]: lastOrNull should give us null if there's nothing ... let's do a fancy kotlin oneliner :sunglasses:

**Review by andr-dev — APPROVED** [review]
Nice!

File: android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt (hunk 1 above)
Comment (leeminkyu16) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt:135]: Huh? Not really sure how I'm supposed to make this into a fancier Kotlin oneliner?

**Review by abhaykshar — APPROVED** [review]

**Review by abhaykshar — APPROVED** [review]
re-approved!

**Pull request merged**